firefox-ios/Client/Frontend/Settings/ClearPrivateDataTableViewController.swift

```diff
@@ -83,6 +83,7 @@ class ClearPrivateDataTableViewController: ThemedTableViewController {
 
         title = .SettingsDataManagementTitle
 
+        tableView.register(cellType: ThemedCenteredTableViewCell.self)
```

Comment:
We don't need to register the cell here as it's already registered in `ThemedTableViewController`.